Say Experimental in Window class

MikeHillberg opened the pull request:
Multi-window support is only supported in _Experimental_, not Preview (or Stable).

This would be better if the word "Experimental" was a link to a page talking about the release channels.

MikeHillberg committed:
Say Experimental in Window class

Multi-window support is only supported in _Experimental_, not Preview (or Stable).

This would be better if the word "Experimental" was a link to a page talking about the release channels.

PRMerger20 commented:
@MikeHillberg : Thanks for your contribution! The author(s) have been notified to review your proposed change.

MikeHillberg commented:
Woops, duplicate: #61

MikeHillberg commented:
Reopening, 61 is an issue not a PR

Karl-Bridge-Microsoft committed:
Update window.md

@MikeHillberg - revised wording for link to the WinApp SDK exp release channel. OK?

PRMerger17 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft committed:
Update window.md

modified phrasing

PRMerger8 commented:
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Karl-Bridge-Microsoft commented:
@MikeHillberg - look good?

Karl-Bridge-Microsoft commented:
I'll merge and we can update again if necessary.
